CodeNow/api
# RUNNABLE_BRANCH_ID in container envs

when you create a container now, we put the instance shorthash in the envs as `RUNNABLE_BRANCH_ID`.

To test:
- switch your elastic URL over to this API server
- use runnable-angular to use this server and create a new instance (make a change to the ports or something)
- do `env` in the container and make sure `RUNNABLE_BRANCH_ID` is there
